docs/messaging/server-integration.md

```diff
@@ -163,6 +163,20 @@ function App() {
 Your application code can then handle messages as you see fit; updating local cache, displaying a [notification](/messaging/notifications)
 or updating UI. The possibilities are endless!
 
+## Signing out users 
+Firebase Cloud Messaging tokens are associated with the instance of the installed app. So reinstalling the app and logging back in with the same user will generate a fresh token. 
```

Comment:
Logging in with the same user will not generate a fresh token, that would be a contradiction of the first (correct) sentence here where you explain that it is tied to the installation, not the user

```suggestion
Firebase Cloud Messaging tokens are associated with the instance of the installed app. By default, only token expiration or uninstalling/reinstalling the app will generate a fresh token.
```